its/src/test/java/com/sonar/it/scanner/msbuild/sonarqube/ExternalIssuesTest.java

```diff
@@ -83,13 +101,15 @@ void externalIssues_CS() {
       assertThat(issues).hasSize(3);
     } else {
       // Not expecting any external issues
-      assertThat(issues).hasSize(2);
+      assertThat(issues).hasSize(3);
```

Comment:
I think the whole if else can be dropped.
We don't run ITs under SQS 7.4.
```suggestion
    assertThat(ruleKeys).containsAll(List.of("external_roslyn:Wintellect004"));
    assertThat(issues).hasSize(3);
```